README.md

```diff
@@ -3,406 +3,217 @@
         <img src="https://raw.github.com/DevExpress/testcafe/master/media/logo.png" alt="testcafe" />
     </a>
 </h1>
+
 <p align="center">
-<a href="https://devexpress.github.io/testcafe">https://devexpress.github.io/testcafe</a>
+<a href="https://ci.appveyor.com/project/DevExpress/testcafe"><img alt="Functional Windows desktop" src="https://ci.appveyor.com/api/projects/status/ftelkyuiji8lyadf?svg=true"></a>
+<a href="https://travis-ci.org/DevExpress/testcafe"><img alt="All Travis tasks (server, client, functional: mobile, macOS, Edge)" src="https://travis-ci.org/DevExpress/testcafe.svg"></a>
+<a href="https://www.npmjs.com/package/testcafe"><img alt="NPM Version" src="https://img.shields.io/npm/v/testcafe.svg" data-canonical-src="https://img.shields.io/npm/v/testcafe.svg" style="max-width:100%;"></a>
 </p>
+
 <p align="center">
-<i>Automated browser testing for the modern web development stack.</i>
+<i>A node.js tool to automate end-to-end web testing.<br/>Write tests in JS or TypeScript, run them and view results.</i>
 </p>
 
 <p align="center">
-<a href="https://ci.appveyor.com/project/DevExpress/testcafe"><img alt="Functional Windows desktop" src="https://ci.appveyor.com/api/projects/status/ftelkyuiji8lyadf?svg=true"></a>
-<a href="https://travis-ci.org/DevExpress/testcafe"><img alt="All Travis tasks (server, client, functional: mobile, macOS, Edge)" src="https://travis-ci.org/DevExpress/testcafe.svg"></a>
-<a href="https://www.npmjs.com/package/testcafe"><img alt="NPM Version" src="https://img.shields.io/npm/v/testcafe.svg" data-canonical-src="https://img.shields.io/npm/v/testcafe.svg" style="max-width:100%;"></a>
+<a href="https://devexpress.github.io/testcafe">https://devexpress.github.io/testcafe</a>
 </p>
 
 ----
 
-TestCafe is a pure node.js end-to-end solution for testing web apps. It takes care of all the stages: starting browsers, running tests, gathering test results and generating reports. TestCafe doesn’t need browser plugins - it works in all popular modern browsers out-of-the-box.
+* **Works on all popular environments**: TestCafe runs on Windows, MacOS and Linux. Supports desktop, mobile, remote and cloud [browsers](https://devexpress.github.io/testcafe/documentation/using-testcafe/common-concepts/browser-support.html). With UI or headless.
+* **5 minutes to set up**: You don’t need WebDriver or any other testing software. Install TestCafe with one command and you’re ready to test: `npm install -g testcafe`
```

Comment:
> 5 minutes to set up
> Install TestCafe with one command and you’re ready to test: `npm install -g testcafe`

Folks, are you experiencing problems with Internet connection speed or something?

Reply:
People very rarely say exact numbers.
While "a few ..." is very common.
This is why it might catch customer's attention, while they might skip a well-known phrase. That's why I insist on "a minute".